packages/wrangler/src/config/validation.ts

```diff
@@ -455,6 +455,16 @@ function normalizeAndValidateEnvironment(
     true
   );
 
+  deprecated(
+    diagnostics,
+    rawEnv,
+    // @ts-expect-error Unable to get DeepKeyOf to accept this string.
```

Comment:
@petebacondarwin any ideas how this could work? It's not critical...

Reply:
The typing is actually doing its job here. `site` is not a property of `RawEnvironment` it is a property of `RawConfig`!
We need to move this `deprecated()` call to the `normalizeAndValidateConfig()` function.

Reply:
aha I see!